template.tpl

```diff
@@ -1708,6 +1726,20 @@ ___TEMPLATE_PARAMETERS___
       }
     ]
   },
+  {
+    "type": "TEXT",
+    "name": "redditAdsInstanceName",
```

Comment:
Do we need a field for PinterestAdsInstanceName too?

Reply:
> Do we need a field for PinterestAdsInstanceName too?

good catch, but no, bc pinterest isn't yet configured for multi-config on backend